### PR TITLE
Revising DirectQuery Notebooks

### PR DESCRIPTION
**Pull request by andrewyang94**
This PR updates the instructions and screenshots for setting up Snowflake and Clickhouse in their respective DirectQuery notebooks. In addition, the code in the notebooks have been upgraded to reflect `0.8.14` syntax.

### CODE REVIEW

File: 02-technical-guides/directquery-vector/main.ipynb [hunk 1]
```diff
@@ -5,30 +5,29 @@
    "id": "a6806a84-d79b-4ef5-93bc-8e0f77f3b1c9",
```
Comment (andrewyang94) [inline 02-technical-guides/directquery-vector/main.ipynb:5]: <p>I can't remember if we decided that we would be adhering to sentence casing moving forward. If this is the case, I can revert this change.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/atoti/atoti/pull/900/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

File: 02-technical-guides/virtual-hierarchies/main.ipynb [hunk 2]
```diff
@@ -11,53 +11,80 @@
     "tags": []
```
Comment (andrewyang94) [inline 02-technical-guides/virtual-hierarchies/main.ipynb:11]: <p>I can't remember if we decided that we would be adhering to sentence casing moving forward. If this is the case, I can revert this change.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/atoti/atoti/pull/900/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (andrewyang94) [inline 02-technical-guides/virtual-hierarchies/main.ipynb:11]: <p>Took out...</p><p> </p><p>"The trade-off between a smaller memory footprint and faster commit times versus offering a slightly reduced set of features is definitely worth it."</p><p> </p><p>as this was very opinionated.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/atoti/atoti/pull/900/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

File: 02-technical-guides/directquery-vector/main.ipynb (hunk 1 above)
Comment (andrewyang94) [reply 02-technical-guides/directquery-vector/main.ipynb:5]: Confirmed that we are moving forward with sentence casing, thus I removed this change.

File: 02-technical-guides/virtual-hierarchies/main.ipynb (hunk 2 above)
Comment (andrewyang94) [reply 02-technical-guides/virtual-hierarchies/main.ipynb]: Confirmed that we are moving forward with sentence casing, thus I removed this change.

**Comment by andrewyang94** [comment]
Note that I've updated the colors of the annotations for the images (to `#00ADE6`) but S3 doesn't seem to be reflecting the up-to-date change just yet.

File: 02-technical-guides/directquery-vector/main.ipynb (hunk 1 above)
Comment (printhellohetal) [inline 02-technical-guides/directquery-vector/main.ipynb:5]: <p>This screenshot seems to be out of date.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/atoti/atoti/pull/900/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

**Review by printhellohetal — APPROVED** [review]
I think the screenshot where I left a comment should be updated. However, no need for me to re-review once that is updated.

**Pull request merged**